packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -51,97 +46,78 @@ describe('Model > SubjectStore', function () {
 
     describe('with less than three subjects in the queue', function () {
       let populateSpy
+      let rootStore
 
       before(function () {
+        rootStore = setupStores()
         populateSpy = sinon.spy(rootStore.subjects, 'populateQueue')
-        while (rootStore.subjects.resources.size > 2) {
-          rootStore.subjects.advance()
-        }
       })
 
       after(function () {
         rootStore.subjects.populateQueue.restore()
+        rootStore = null
       })
 
       it('should request more subjects', function () {
+        while (rootStore.subjects.resources.size > 2) {
+          rootStore.subjects.advance()
+        }
         expect(populateSpy).to.have.been.calledOnce()
       })
     })
 
     describe('after emptying the queue', function () {
+      let rootStore
       before(function () {
         sinon.stub(clientStub.panoptes, 'get').callsFake(() => Promise.resolve({ body: [] }))
-        while (rootStore.subjects.resources.size > 0) {
-          rootStore.subjects.advance()
-        }
+        rootStore = setupStores()
       })
 
       after(function () {
         clientStub.panoptes.get.restore()
+        rootStore = null
       })
 
       it('should leave the active subject empty', function () {
+        while (rootStore.subjects.resources.size > 0) {
+          rootStore.subjects.advance()
+        }
         expect(rootStore.subjects.resources.size).to.equal(0)
         expect(rootStore.subjects.active).to.be.undefined()
       })
     })
   })
 
-  describe('Views > isThereMetadata', function (done) {
-    it('should return false when there is not an active queue subject', function () {
-      rootStore = RootStore.create(
-        { projects: ProjectStore.create(), subjects: SubjectStore.create(), workflows: WorkflowStore.create() },
-        { client: {
-          panoptes: {
-            get: () => {
-              return Promise.resolve({
-                body: { subjects: [] }
-              })
-            }
-          }
-        }
-        }
-      )
+  describe('Views > isThereMetadata', function () {
+    let rootStore
+    afterEach(function () {
+      rootStore = null
```

Comment:
Instead of resetting `rootStore` between tests, you could use `const rootStore = setupStores()` inside the tests and not have to worry about accidentally changing the mock for one test inside a previous test.